pkg/kubelet/apis/deviceplugin/v1alpha/api.pb.go

```diff
@@ -135,6 +135,8 @@ type Device struct {
 	ID string `protobuf:"bytes,1,opt,name=ID,json=iD,proto3" json:"ID,omitempty"`
 	// Health of the device, can be healthy or unhealthy, see constants.go
 	Health string `protobuf:"bytes,2,opt,name=health,proto3" json:"health,omitempty"`
+	// Attributes of the device. eg: {"version": "1.0"}
+	Attributes map[string]string `protobuf:"bytes,3,rep,name=attributes" json:"attributes,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
```

Comment:
what are these information come from. One use case in my mind is querying nvidia-plugins API. Am I right?

Reply:
Numa manager can also use this:
https://github.com/kubernetes/community/pull/1680#discussion_r163776284

Reply:
The overall design for device topology awareness is not yet finalized. Let's hold off on this PR for now.